cliquet/initialization.py

```diff
@@ -48,6 +48,9 @@ def setup_version_redirection(config):
     redirect_enabled = settings['cliquet.version_prefix_redirect_enabled']
     version_prefix_redirection_enabled = asbool(redirect_enabled)
 
+    route_prefix = config.route_prefix
+    config.add_settings(route_prefix=route_prefix)
```

Comment:
I think you are right, we should add the route_prefix to the registry rather than adding it in the settings.